README.md

```diff
@@ -1,3 +1,5 @@
+THIS IS A FAKE LINE IN THE README SO WE DEMO PRs
```

Comment:
Maybe we shouldn't have this line

Reply:
Yes, I agree with you, on further reflection.  Thanks.  I'll delete it.